tests/example_files.yaml

```diff
@@ -49,21 +49,21 @@
 - local_filename: NewJersey.pdf
   url: https://github.com/py-pdf/pypdf/files/12090692/New.Jersey.Coinbase.staking.securities.charges.2023-0606_Coinbase-Penalty-and-C-D.pdf
 - local_filename: tika-952445.pdf
-  url: https://corpora.tika.apache.org/base/docs/govdocs1/952/952445.pdf
+  url: https://github.com/user-attachments/files/18382348/tika-952445.pdf
 - local_filename: tika-921632.pdf
-  url: https://corpora.tika.apache.org/base/docs/govdocs1/921/921632.pdf
+  url: https://github.com/user-attachments/files/18382354/tika-921632.pdf
 - local_filename: tika-976970.pdf
-  url: https://corpora.tika.apache.org/base/docs/govdocs1/976/976970.pdf
+  url: https://github.com/user-attachments/files/18382397/tika-976970.pdf
 - local_filename: tika-914102.pdf
-  url: https://corpora.tika.apache.org/base/docs/govdocs1/914/914102.pdf
+  url: https://github.com/user-attachments/files/18381687/tika-914102.pdf
 - local_filename: iss1737.pdf
   url: https://github.com/py-pdf/pypdf/files/11068604/tt1.pdf
 - local_filename: issue-1801.pdf
   url: https://github.com/py-pdf/pypdf/files/11250359/test_img.pdf
 - local_filename: tika-924546.pdf
-  url: https://corpora.tika.apache.org/base/docs/govdocs1/924/924546.pdf
+  url: https://github.com/user-attachments/files/18381697/tika-924546.pdf
```

Comment:
there is some duplicates in this file that could be cleaned up. no urgence

Reply:
Should be fixed now as well.